ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.15-nightly-x86.yaml

```diff
@@ -0,0 +1,54 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-azure-415
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.15"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: control-plane-120nodes
+  cron: 0 12 * * 3
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "120"
+      COMPUTE_NODE_TYPE: Standard_D4s_v3
+      CUSTOM_AZURE_REGION: centralus
+      SIZE_VARIANT: large
```

Comment:
I see SIZE_VARIANT always picks v3 instance generation, can we explicitly set the control-plane to work with v5 instance generation?

Reply:
@krishvoor @qiliRedHat I bought up the discussion of instance types during recent calls. From what I gather we want to make sure it is similar sizing as AWS and Rosa clusters. If we need to adjust workloads for those instance types, that's what we will do instead of changing instance types.

Reply:
> @krishvoor @qiliRedHat I bought up the discussion of instance types during recent calls. From what I gather we want to make sure it is similar sizing as AWS and Rosa clusters. 

@svetsa-rh The instance sizes have to be aligned with Self-Managed/ROSA, however my comments were towards
Instance Family type `Standard_D32s_v3` vs `Standard_D32s_v5` which allows us to tap into the latest generation of offering on Azure MarketPlace.

Reply:
>If we need to adjust workloads for those instance types, that's what we will do instead of changing instance types

The config refits/resizes will have to go via 
https://issues.redhat.com/browse/PERFSCALE-2360 
https://issues.redhat.com/browse/PERFSCALE-2361